sci-libs/ifcopenshell/ifcopenshell-0.8.0_p20241021.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit cmake python-single-r1
+
+COMMIT=5fa5b284848ae8f7551c7c56dccf5c3362ab2abf
```

Comment:
I am a bit confused about the version numbers here, why are these snapshots? Upstream does seem to have some tags.